README.md

```diff
@@ -1 +1,35 @@
-# leaderboard
\ No newline at end of file
+# LeaderBoard
+This application is used to store and retrive the name and socre of a player 
+
+## Leaderbaord screen shot
+<img width="788" alt="leaderboardschreenshot" src="https://user-images.githubusercontent.com/22744775/181681593-1bbcebfc-b6ee-454d-b0e3-5bf4dc327b9b.PNG">
+
+## Page Link 
+https://hamayounali.github.io/leaderboard/
+
+# Setup
+Clone this repository
+
+$ git clone git@github.com:Hamayounali/leaderboard.git
+$ cd leaderboard
+
+Run the live server or open index.html with your browser
```

Comment:
- [OPTIONAL] I suggest adding more instructions about how to run the project, for example, you can add `npm install` to install all dependencies and then `npm run start` to run the project. 👌

Reply:
Noted, Thank you